.travis.yml

```diff
@@ -65,6 +65,11 @@ matrix:
       go: 1.15.x
       script:
         - make build-backend
+    - name: "Verify go unit tests"
```

Comment:
I think this should be called "Run go unit tests"? The tests are not _subject_ to verification, the tests are _doing_ the verification, no?

Reply:
done